pom.xml

```diff
@@ -66,8 +66,7 @@ THE SOFTWARE.
         <revision>2.22</revision>
         <changelist>-SNAPSHOT</changelist>
         <java.level>8</java.level>
-        <jenkins.version>2.121.1</jenkins.version>
```

Comment:
You should always specify an explicit `jenkins.version`. Do _not_ attempt to inherit from the parent POM; that is merely a minimum recommended version.

Reply:
Ok, reverted this change.